Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby "2.3.3"
+ruby "2.4.1"
```

Comment:
We can't make changes to the Ruby version right now because it breaks the bin build process. This code looks fine to run on 2.3.3, Gemfile and Gemfile.lock shouldn't be tracked in the commit. I think I can remove them from the PR, let me test.

Reply:
Roger that.